comet/scrapers/torbox.py

```
@@ -0,0 +1,36 @@
+            torrents.append(
+                {
+                    "title": torrent["raw_title"],
+                    "infoHash": torrent["hash"],
+                    "fileIndex": None,
+                    "seeders": torrent["last_known_seeders"],
+                    "size": torrent["size"],
+                    "tracker": f"TorBox|{torrent['tracker']}",
+                    "sources": extract_trackers_from_magnet(torrent["magnet"]),
+                }
+            )
+    except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)